### PR TITLE
added clear message for missing releaseNotes

### PR DESCRIPTION
**Pull request by shaniacht1**

### CODE REVIEW

File: Tests/release_notes.py [hunk 1]
```diff
@@ -57,14 +59,21 @@ def generateRN(self):
                 modifiedStr = ""
                 for rawContent in self.modifiedStore:
                     cnt = self.loadData(rawContent)
-                    modifiedStr += self.modifiedReleaseNotes(cnt)
+                    ans = self.modifiedReleaseNotes(cnt)
+                    if ans is None:
+                        print_error(cnt["name"] + " yml is missing releaseNotes entry")
```
Comment (dorsha) [inline Tests/release_notes.py]: it can be also json.. (reports/reputations)

File: setContentDescriptor.sh [hunk 2]
```diff
@@ -6,4 +6,4 @@ git diff  --diff-filter=D $2 > delete-changelog.txt
 ASSETID=$1
 VERSION=$3
 
-python release_notes.py  $VERSION changelog.txt delete-changelog.txt $ASSETID
\ No newline at end of file
+python Tests/release_notes.py  $VERSION changelog.txt delete-changelog.txt $ASSETID
```
Comment (aviadl) [inline setContentDescriptor.sh]: Why did you move it to tests ?

Comment (shaniacht1) [inline setContentDescriptor.sh]: I wanted to use the test_utils file that's in Tests + id made more sense since it is a test

**Comment by shaniacht1** [comment]
@aviadl In every possible way

File: setContentDescriptor.sh (hunk 2 above)
Comment (aviadl) [inline setContentDescriptor.sh]: It is not a test
It is a tool that is part of our build process, it  creates our release notes

Comment (shaniacht1) [inline setContentDescriptor.sh]: ok. moved it back

File: Tests/__init__.py [hunk 3]
```diff
@@ -0,0 +1 @@
+#this file enables usage for other files from different directories
```
Comment (asafshen) [inline Tests/__init__.py]: Are we sure that this this file is required?

File: __init__.py [hunk 4]
```diff
@@ -0,0 +1 @@
+#this file enables usage for other files from different directories
```
Comment (asafshen) [inline __init__.py]: Are we sure that this this file is required?

File: release_notes.py [hunk 5]
```diff
@@ -43,6 +45,7 @@ def loadData(self, data):
 
     def generateRN(self):
         res = ""
+        error_count = 0
```
Comment (asafshen) [inline release_notes.py]: Very not critical - but if you only want to know if there was an error - you can use boolean value instead

(And give it more meaningful name - such has `missingReleseNotes`

File: release_notes.py [hunk 6]
```diff
@@ -293,14 +302,21 @@ def main(argv):
...
+        sys.exit(1)
     version = argv[0]
     assetId = argv[3]
     createContentDescriptor(version, assetId, res)
 
 
 if __name__ == "__main__":
-   main(sys.argv[1:])
\ No newline at end of file
+   main(sys.argv[1:])
```
Comment (asafshen) [inline release_notes.py]: please add newline in the end of the file

File: release_notes.py [hunk 7]
```diff
@@ -293,14 +302,21 @@ def main(argv):
         createFileReleaseNotes(file, argv[2])
 
     res = ""
+    error_count = 0
```
Comment (asafshen) [inline release_notes.py]: same note as above (boolean + naming)

**Comment by dorsha** [comment]
@shaniacht1 / @asafshen - what about this one?

**Comment by shaniacht1** [comment]
@dorsha I will go over it after EndGame

**Comment by asafshen** [comment]
@shaniacht1 if you feel that is ready for merge - please do 😃
